generate-schema.js

```diff
@@ -0,0 +1,8 @@
+const convert = require('joi-to-json-schema')
+const fs = require('fs')
+const { schema } = require('./lib/schema')
+
+const theSchema = convert(schema())
+// theSchema.$id = ''
```

Comment:
This comment can be removed, I assume? :)

Reply:
I assume we have some fixes so we can publish the schema. That's why I left it in 😊